concourse/client/__init__.py

```diff
@@ -76,3 +76,11 @@ def from_cfg(concourse_cfg: ConcourseConfig, team_name: str, verify_ssl=False):
         passwd=password,
     )
     return concourse_api
+
+
+@functools.lru_cache()
+@ensure_annotations
+def from_cfg_name(concourse_cfg_name: str, team_name: str, verify_ssl: bool=False):
```

Comment:
I think this should go into `ccc.concourse`

Reply:
also, I would prefer not to expose `verify_ssl` as a parameter. it is already configurable via cc-config (and even that could be considered a bad idea). Let us not make it too easy to disable tls-validation